nginx.conf

```diff
@@ -18,11 +18,11 @@ http {
     include       mime.types;
     default_type  application/octet-stream;
 
-    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+    log_format  main  '$image_build_date $remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
-    #access_log  /var/log/nginx/access.log  main;
+    access_log logs/access.log main;
```

Comment:
does this work when the image is run as non-root? Also are the logs still present in STDOUT of the container?

Reply:
otherwise use `/dev/null` as output file if you really need a file